File: TCS_MOCK_VITA/Github_Repository_Files/RoundB_B.py
def score(n):
    lsb=10
    msb=-1
    temp=n
    while temp>0:
        lsb=min(lsb,temp%10)
        temp//=10
    temp=n
    while temp>0:
        msb=max(msb,temp%10)
        temp//=10
    ans=msb*11+lsb*7
    d1=ans%10
    ans//=10
    d2=ans%10
    ans=d2*10+d1
    return ans
def msb(n):
    n//=10
    return n
n=int(input())
a=list(map(int,input().split()))
b=[score(a[i]) for i in range(n)]
ans=0
for m in range(10):
    count1=0
    for i in range(0,n,2):
        if msb(b[i])==m:
            count1+=1
    count2=0
    for i in range(1,n,2):
        if msb(b[i])==m:
            count2+=1
    if count1>2 or count2>2:
        ans+=2
    elif count1==2 or count2==2:
        ans+=1
print(ans)

File: TCS_MOCK_VITA/Github_Repository_Files/RoundB_B.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
#include<math.h>
#define is ==
#define and &&
#define or ||
#define not !
#define max(a,b) a>b?a:b
#define min(a,b) a<b?a:b
typedef long long int ll;
int get_bit(const int n)
{
    int small=10;
    int large=-1;
    int temp=n;
    while(temp>0)
    {
        small=min(small,temp%10);
        temp/=10;
    }
    temp=n;
    while(temp>0)
    {
        large=max(large,temp%10);
        temp/=10;
    }
    int ans=large*11+small*7;
    int digit1=ans%10;
    ans/=10;
    int digit2=ans%10;
    ans=(digit2*10+digit1);
    return ans;
}
int msb(int n)
{
    n/=10;
    return n;
}
int main()
{
    int n;
    scanf("%d",&n);
    int a[502];
    for(int i=0;i<n;i++)
        scanf("%d",&a[i]);
    int bit[502];
    for(int i=0;i<n;i++)
    {
        bit[i]=get_bit(a[i]);
        //printf("%d ",bit[i]);
    }
    int ans=0;
    for(int m=0;m<10;m++)
    {
        int count1=0;
        for(int i=0;i<n;i+=2)
        {
            if(msb(bit[i])==m)
                count1++;
        }
        int count2=0;
        for(int i=1;i<n;i+=2)
        {
            if(msb(bit[i])==m)
                count2++;
        }
        if(count1>2 or count2>2)
            ans+=2;
        else if(count1==2 or count2==2)
            ans+=1;
    }
    printf("%d",ans);
    return 0;
}